Bio/ExPASy/Enzyme.py

```diff
@@ -55,7 +55,7 @@ def read(handle):
 
 
 class Record(dict):
-    """\
+    r"""\
```

Comment:
While you are editing this bit of the code, can you remove the trailing slash and use:

``` python
class Record(dict):
    r"""Holds from an ExPASy ENZYME record as a Python dictionary.

    Each record contains the following keys:
    ...
```